Cargo.lock

```diff
@@ -5380,7 +5425,7 @@ dependencies = [
  "futures-executor",
  "futures-util",
  "glob",
- "opentelemetry",
+ "opentelemetry 0.27.1",
```

Comment:
we've ended up with some packages relying on the old opentelemetry. do we need to update some other otel packages so they are all in line?